### PR TITLE
Release 2.0 

### PR DESCRIPTION
**Pull request by pun-ky**
WIP

### CODE REVIEW

File: wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/WireMockStubs.java [hunk 1]
```diff
@@ -189,9 +178,21 @@ public WireMockServlet getServlet() {
...
+      name = "Request journal",
+      description = "Enable the request journal, which records incoming requests for later verification. Protects against reserving too much memory."
+    )
+    boolean requestJournalEnabled() default false;
+
+    @AttributeDefinition(
+      name = "Max Request Journal Entries",
+      description = "Set maximum number of entries in request journal (if enabled). When this limit is reached oldest entries will be discarded. 0 means no limits."
+    )
+    int requestJournalMaxSize() default 0;
```
Comment (pun-ky) [inline wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/WireMockStubs.java:196]: enabled = false is good default for safety reasons. however maxSize = 0 is not safe default when enabled = true. So I recommend to propose some reasonable value > 0 here.

File: wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/servlet/handler/AdminHandler.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    }
+  }
+
+  private String prepareErrorMessage(Exception e) {
+    return String.format(ERROR_TEMPLATE, prepareErrorMessage(), e.getMessage());
+  }
+
+  private String prepareErrorMessage() {
+    if (isGetMappingsUrl()) {
+      return "One of your mapping definition use dynamic template value and cannot be deserialized";
```
Comment (pun-ky) [inline wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/servlet/handler/AdminHandler.java:51]: outdated?:D

Comment (marcinkp) [reply wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/servlet/handler/AdminHandler.java:51]: you are right :)

File: wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/WireMockStubs.java (hunk 1 above)
Comment (marcinkp) [reply wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/WireMockStubs.java]: set 200 as default

**Review by marcinkp — APPROVED** [review]

**Pull request merged**